crates/ruff/src/rules/flake8_variables_names/settings.rs

```diff
@@ -0,0 +1,44 @@
+//! Settings for the `flake8-variables-names` plugin.
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use ruff_macros::{CacheKey, ConfigurationOptions};
+
+#[derive(
+    Debug, PartialEq, Eq, Serialize, Deserialize, Default, ConfigurationOptions, JsonSchema,
+)]
+#[serde(
+    deny_unknown_fields,
+    rename_all = "kebab-case",
+    rename = "Flake8VariablesNamesOptions"
+)]
+pub struct Options {
+    #[option(
+        default = "false",
+        value_type = "bool",
+        example = "use-varnames-strict-mode = true"
+    )]
+    /// Whether to use strict settings for the allow/block lists
+    pub use_varnames_strict_mode: Option<bool>,
```

Comment:
We can probably just shorten this to `strict`, since unlike in Flake8, this is scoped under the plugin settings, which makes it clear that it applies to `flake8-variable-names`.

Reply:
Makes sense to me!